[ASM] - EXPANDR -  1578 (#24380)
demisto/content: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_1.md

#### Playbooks
##### New: Cortex ASM - Decision

> - Created a new playbook that returns "Remediation Action" options based on meeting "Automated Remediation Requirements" (https://github.com/demisto/content/tree/master/Packs/CortexAttackSurfaceManagement#automated-remediation-requirements) as well as whether ServiceNowV2 integration is setup.

Reviewer: ```suggestion
Created a playbook that returns "Remediation Action" options based on meeting "Automated Remediation Requirements" (https://github.com/demisto/content/tree/master/Packs/CortexAttackSurfaceManagement#automated-remediation-requirements) as well as whether ServiceNowV2 integration is set up.
```

##### New: Cortex ASM - Remediation Path Rules

> - Created a new playbook that returns "Remediation Action" options based return from Remediation Path Rules API, or defaults to data collection task options from "Cortex ADM - Decision" subplaybook.

Reviewer: ```suggestion
Created a playbook that returns "Remediation Action" options based on the return from the Remediation Path Rules API, or defaults to data collection task options from the "Cortex ADM - Decision" sub- playbook.
```

##### Cortex ASM - ASM Alert

> - Updated the playbook to include the new "Cortex ASM - Remediation Path Rules" subplaybook to include the remediation path logic.

Reviewer: ```suggestion
- Updated the playbook to include the new "Cortex ASM - Remediation Path Rules" sub-playbook to include the remediation path logic.
```

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: ```suggestion
- Remove the *marketplaces* field from the yml file.
```

##### Cortex ASM - CMDB Enrichment

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: ```suggestion
- Remove the *marketplaces* field from the yml file.
```

##### Cortex ASM - Detect Service

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: ```suggestion
Remove the *marketplaces* field from the yml file.
```

##### Cortex ASM - Enrichment

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: Removed the *marketplaces* field from the yml file.

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: ```suggestion
Removed the *marketplaces* field from the yml file.
```

##### Cortex ASM - GCP Enrichment

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: ```suggestion
Removed the *marketplaces* field from the yml file.
```

##### Cortex ASM - Remediation Guidance

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: ```suggestion
Removed the *marketplaces* field from the yml file.
```

##### Cortex ASM - Remediation

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: ```suggestion
Removed the *marketplaces* field from the yml file.
```

##### Cortex ASM - Tenable.io Enrichment

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: ```suggestion
Removed the *marketplaces* field from the yml file.
```

#### Scripts
##### GenerateASMReport

> - Updated the GenerateASMReport to include the new "ASM - Remediation Path Rule" field.

Reviewer: ```suggestion
- Updated the GenerateASMReport to include the new *ASM - Remediation Path Rule* field.
```

##### New: RemediationPathRuleEvaluation

> - Added the RemediationPathRuleEvaluation automation that checks for a matching remediation path rule (Xpanse only).

Reviewer: ```suggestion
- Added the **RemediationPathRuleEvaluation** automation that checks for a matching remediation path rule (Xpanse only).
```

#### Incident Fields

> - **ASM - Remediation Path Rule**

Reviewer: ```suggestion
**ASM - Remediation Path Rule**
```

#### Integrations
##### Cortex Attack Surface Management

> - Added the **asm-list-remediation-rule** command that returns list of remediation path rules (Xpanse only).

Reviewer: ```suggestion
- Added the ***asm-list-remediation-rule*** command that returns a list of remediation path rules (Xpanse only).
```

> - Updated the yml file to no longer include 'marketplaces' field.

Reviewer: ```suggestion
- Removed the *marketplaces* field from the yml file.
```
